plugins/generator-1.17.1/forge-1.17.1/procedures/world_data_get_indirect_power.java.ftl

```diff
@@ -0,0 +1 @@
+((world instanceof Level)?((Level) world).getBestNeighborSignal(new BlockPos((int)${input$x},(int)${input$y},(int)${input$z})):0)
```

Comment:
I would use new J16 casting here for it to be more clean, same with other procedures in the PR where this applies